lib/quantizeAttributes.js

```diff
@@ -0,0 +1,281 @@
+'use strict';
+var Cesium = require('cesium');
+
+var defined = Cesium.defined;
+
+var addExtensionsUsed = require('./addExtensionsUsed');
+var byteLengthForComponentType = require('./byteLengthForComponentType');
+var getAccessorByteStride = require('./getAccessorByteStride');
+var numberOfComponentsForType = require('./numberOfComponentsForType');
+var removeUnusedVertices = require('./removeUnusedVertices');
+
+module.exports = quantizeAttributes;
+
+/**
+ * Quantizes attributes in the provided glTF using the WEB3D_quantized_attributes extension.
+ * If options is undefined, all quantizable attributes will be quantized with the default options.
+ * If options.attributes is undefined, all quantizable attributes will be quantized with the provided options.
+ * If options.precision is undefined, the decodeMatrix be written in its full form.
+ *
+ * @param {Object} [gltf] A javascript object holding a glTF hierarchy.
+ * @param {Object} [options=undefined] Defines more specific quantization behavior.
+ * @param {Object} [options.attributes=undefined] Defines specific attributes to be quantized. This should be as arranged as meshId -> primitiveIndex -> [semantic_0, semantic_1, ... semantic_N]
```

Comment:
See [CesiumMilkTruck-mismatch](https://github.com/lasalvavida/cesium/blob/bf81d91d104fa3d0d1d74ab6a5451070c1f3c75f/Specs/Data/Models/WEB3DQuantizedAttributes/CesiumMilkTruck-Mismatch-Quantized.gltf)

Just the POSITIONS and NORMALS on one primitive in a mesh are quantized, and just the TEXCOORD on a primitive in a different mesh is quantized. 

In practice, I agree, you generally just want to quantize everything or a specific semantic, but I needed those options to help generate potentially breaking test cases.